.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
You could have this run on our own CI servers instead I think. See [this example](https://github.com/aurora-is-near/aurora-engine/blob/master/.github/workflows/tests.yml#L13) from the Aurora Engine. This would likely mean you don't have to install all the dependencies each time.

Reply:
Is it enough to just write that i run it in self-hosting machine and github will automatically find auroras machine or I should write info about self-hosting machine

Reply:
As far as I understand it you do not need to give any details about the self-hosted machine; it will just work for any repo under the `aurora-is-near` organization.